fdbserver/CommitProxyServer.actor.cpp

```diff
@@ -1119,6 +1197,14 @@ ACTOR Future<Void> postResolution(CommitBatchContext* self) {
 		wait(Future<Void>(Never()));
 	}
 
+	if (SERVER_KNOBS->ENABLE_VERSION_VECTOR_TLOG_UNICAST && self->metadataMutationFromProxy &&
+	    !self->hasMetadataMutation) {
```

Comment:
What happens if `self->hasMetadataMutation` is true? Then all metadata mutations are committed. Will TPCV vector be correct?

Reply:
if `hasMetadataMutation` is true we operate like normal FDB. We write to all tLogs and get the TPCV for each.